### PR TITLE
Zariski coverage on CommRing^op

### PR DESCRIPTION
**Pull request by mzeuner**
Finishing the draft PR (https://github.com/agda/cubical/pull/1081) with @MatthiasHu.

### CODE REVIEW

File: Cubical/Categories/Site/Instances/ZariskiCommRing.agda [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+pullbackStability zariskiCoverage {c = A} um {d = B} φ =
+  ∣ pullbackUniModVec φ um , (λ i → ∣ i , ψ i , RingHom≡ (sym (ψComm i)) ∣₁) ∣₁
+  where
+  open UniModVec
+  module A = InvertingElementsBase A
+  module B = InvertingElementsBase B
+  module AU = A.UniversalProp
+  module BU = B.UniversalProp
+
+  ψ : (i : Fin (um .n)) → CommRingHom A.R[1/ um .f i ]AsCommRing B.R[1/ φ $r um .f i ]AsCommRing
```
Comment (mzeuner) [inline Cubical/Categories/Site/Instances/ZariskiCommRing.agda:103]: Unfortunately, `--lossy-unification` is needed to both show that `ψ` has the above type and fills the goal where it's used. I tried to play around with `opaque`, but to no avail.

Any ideas anyone?

File: Cubical/Algebra/CommRing/Localisation/InvertingElements.agda [hunk 2]
```diff
@@ -279,6 +279,40 @@ module InvertingElementsBase (R' : CommRing ℓ) where
                 PT.rec (PisProp s) λ (n , p) → subst P (sym p) (base n)
 
 
+
+ -- A more specialized universal property.
+ -- (Just ask f to become invertible, not all its powers.)
+ module UniversalProp (f : R) where
```
[the remark this answers is not in the document]
Comment (mzeuner) [reply Cubical/Algebra/CommRing/Localisation/InvertingElements.agda:285]: This module also exports the basic things related to the universal property of general localization publicly. I would argue that this is more convenient to work with.

File: Cubical/Algebra/Monoid/BigOp.agda [hunk 3]
```diff
@@ -57,3 +57,17 @@ module MonoidBigOp (M' : Monoid ℓ) where
 
  bigOpSplit++ {n = zero} V W = sym (·IdL _)
  bigOpSplit++ {n = suc n} V W = cong (V zero ·_) (bigOpSplit++ (V ∘ suc) W) ∙ ·Assoc _ _ _
+
+
+module _ {M : Monoid ℓ} {M' : Monoid ℓ'} (φ : MonoidHom M M') where
+  module M = MonoidBigOp M
+  module M' = MonoidBigOp M'
```
Comment (MatthiasHu) [inline Cubical/Algebra/Monoid/BigOp.agda:64]: These two should probably be private. (`module ... = ...` statements are public by default, as opposed to `open ...` statements.)

**Review by MatthiasHu — APPROVED** [review]
This PR is ready to be merged.

**Pull request merged**